netdata-installer.sh

```diff
@@ -1022,6 +1024,72 @@ install_go() {
 
 install_go
 
+should_install_ebpf() {
+  if [ "${NETDATA_ENABLE_EBPF:=0}" -ne 1 ]; then
```

Comment:
Should we also add a condition to verify the kernel version here? Case the kernel is previous `4.11`, we won't have conditions, independent of the users say they want to use.
I saw that you added a condition in the next function to this, but the name of this function says me that we could bring the test from `install_ebpf` to here.
This is only a suggestion.

Reply:
Sorry yes you are right; it was late at night and I _actually_ meant to have the check here :) I will move it!